roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml

```diff
@@ -24,19 +24,21 @@
   with_items:
   - hawkular-metrics.pwd
   - hawkular-metrics.htpasswd
-  - hawkular-jgroups-keystore.pwd
   changed_when: no
 
 - set_fact:
     pwd_files: "{{pwd_files | default({}) | combine ({item.item: item.stat}) }}"
   with_items: "{{pwd_file_stat.results}}"
   changed_when: no
 
-- name: generate password for hawkular metrics and jgroups
+- name: generate password for hawkular metrics
   local_action: copy dest="{{ local_tmp.stdout}}/{{ item }}.pwd" content="{{ 15 | oo_random_word }}"
   with_items:
   - hawkular-metrics
-  - hawkular-jgroups-keystore
+
+- name: generate password for hawkular metrics
+  shell: openssl rand -base64 512 | tr -dc A-Z-a-z-0-9 | head -c 17
```

Comment:
we have a filter module that we can use instead of shell, `oo_random_word`. Can this be updated to use that instead?
I believe the usage would be `"{{ 17 | oo_random_word }}"`